src/main/interface/cli.c

```diff
@@ -4318,6 +4486,10 @@ static void printConfig(char *cmdline, bool doDiff)
     if (dumpMask & DUMP_RATES) {
         cliDumpRateProfile(systemConfig_Copy.activeRateProfile, dumpMask);
     }
+
+    if (dumpMask & DUMP_OSD_PROFILE) {
```

Comment:
`all` should include all OSD profiles as well, otherwise it becomes useless for backup / restore.

Reply:
Hope I understood your comment correctly:  It does include it, dump or dump all lists all OSD elements with their profiles,i.e. for vbat it looks like this:
set osd_vbat_pos = 399
osdprofile osd_vbat_pos = 1